platform/kube/inject/initializer.go

```diff
@@ -162,29 +162,38 @@ func NewInitializer(restConfig *rest.Config, config *Config, cl kubernetes.Inter
 	return i, nil
 }
 
-func (i *Initializer) initialize(in interface{}, patcher patcherFunc) error {
-	obj, err := meta.Accessor(in)
-	if err != nil {
-		return err
+func modifyRequired(objNamespace string, ignore, exclude, include []string) bool {
+	for _, namespace := range ignore {
+		if namespace == objNamespace {
+			return true
+		}
 	}
-
-	var inject bool
-namespaceSearch:
-	for _, namespace := range i.config.IncludeNamespaces {
+	for _, namespace := range exclude {
+		if namespace == objNamespace {
+			return true
+		}
+	}
+	for _, namespace := range include {
 		if namespace == v1.NamespaceAll {
-			inject = true
-			break namespaceSearch
-		} else if namespace == obj.GetNamespace() {
+			return true
```

Comment:
Should it also return true if it is on the included namespace list?  I asked because the PR discription also said excluded and ignored namespaces.

Reply:
Yes, line 179 checks to see if the object's namespace matches the current value in the list of included namespaces.